src/rocks-command/rocks/commands/report/host/vm/config/__init__.py

```diff
@@ -194,6 +194,9 @@ def getInterfaces(self, node):
 			returnxml.append("    <interface type='bridge'>")
 			returnxml.append("      <source bridge='%s'/>" % bridged_inter["device"])
 			returnxml.append("      <mac address='%s'/>" % bridged_inter["mac"])
+			returnxml.append("      <filterref filter='clean-traffic'>")
```

Comment:
This looks too easy... Can you show us what did you do to test? Can you summarize how this is implemented in the host (i.e. dynamic iptables rules?). If iptables is restarted on the physical host what happens to this configuration? Do we need to add firewall rules to rocks and/or do an iptables-save on the physical host to prevent loss of this setting?